.github/workflows/main.yml

```diff
@@ -122,13 +122,12 @@ jobs:
         working-directory: extensions/ql-vscode
         if: matrix.os == 'ubuntu-latest'
         run: |
-          CODEQL_PATH=$GITHUB_WORKSPACE/codeql-home/codeql/codeql npm run test
```

Comment:
So these tests are not using the CLI? Should we remove it from being installed in the step above?

Reply:
They may have at one point in the past, but not any more. I will remove the section.